.github/workflows/gem5.yml

```
@@ -217,45 +146,43 @@ jobs:
        github.event.pull_request.head.repo.full_name == github.repository)
     runs-on: [self-hosted, open]
     continue-on-error: false
-    name: XS-GEM5 - Check memory corruption
+    timeout-minutes: 180
+    name: XS-GEM5 - Check memory errors and leaks
+    env:
+      MEMORY_CHECK_OUTPUT_DIR: ${{ runner.temp }}/gem5-valgrind-${{ github.run_id }}-${{ github.run_attempt }}
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)